contrib/debian/netdata.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
there is no need to duplicate the service, please try to consolidate with what we have at /system